src/main/helpers/ps.ts

```diff
@@ -1,12 +1,32 @@
 import psList from "ps-list";
-import { tasklist } from "tasklist";
+import path from "node:path";
+import childProcess from "node:child_process";
+import { promisify } from "node:util";
 
-export const getProcesses = async () => {
-  if (process.platform === "win32") {
-    return tasklist().then((tasks) =>
-      tasks.map((task) => ({ ...task, name: task.imageName }))
-    );
-  }
+const TEN_MEGABYTES = 1000 * 1000 * 10;
+const execFile = promisify(childProcess.execFile);
+
+export const getProcesses = async (isPackaged: boolean) => {
```

Comment:
Why don't you access `app.isPackaged` from this function directly instead of passing it as an argument? I think you should be good.

Reply:
sure, done